bundles/ranvier-crafting/help/craft.yml

```diff
@@ -1,5 +1,5 @@
 ---
-command: quest
+command: craft <list/create> [category #] [item #]
```

Comment:
This is the wrong way to do it, the purpose of the `command` entry in the helpfile is to do a `CommandManager` lookup to get the `Command.usage` property. So this should not be necessary